automatic/vim/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,50 @@
+﻿$ErrorActionPreference = 'Stop';
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$packageArgs = @{
+        packageName    = $env:ChocolateyPackageName
+        unzipLocation  = $toolsDir
+        url            = 'https://github.com/vim/vim-win32-installer/releases/download/v8.1.2056/gvim_8.1.2056_x86.zip'
```

Comment:
You should embedd vim zip into the package. It has permissive license and is not big.